src/Shared/Components/header/BookmarkIcon.js

```diff
@@ -0,0 +1,80 @@
+/*
+  The Bookmark Icon sits on the Breadcrumb component and passes the legislation props to the icon actions.
+  The icon is only visible on pages that have Legislation that can be bookmarked although the icon is technically "seen" by all the screens that contain the breadcrumb.
+  The icon renders differently depending on whether the legislation is found in the redux.
+*/
+
+import React, { useState, useEffect } from "react";
+import { View } from "react-native";
+import { useDispatch, useSelector } from "react-redux";
+import { addBookmark, removeBookmark } from "../../../Redux/bookmarkSlice";
+import Icon from "../Icon.js";
+import styles from "../../styles";
+import { RoutingItems } from "../../../Data/RoutingLookups";
+
+export const BookmarkIcon = ({
+  title,
```

Comment:
Per the definition of done, can you document what the passing parameters are for and how to use them?

Reply:
Done.